internals/daemon/daemon.go

```diff
@@ -327,10 +396,10 @@ func logit(handler http.Handler) http.Handler {
 				r.URL.Path == "/v1/health")
 		if !skipLog {
 			if strings.HasSuffix(r.RemoteAddr, ";") {
-				logger.Debugf("%s %s %s %s %d", r.RemoteAddr, r.Method, r.URL, t, ww.status())
-				logger.Noticef("%s %s %s %d", r.Method, r.URL, t, ww.status())
+				logger.Debugf("%s %s %s %s %d (transport: %s)", r.RemoteAddr, r.Method, r.URL, t, ww.status(), transport)
```

Comment:
For these logs, I think just saying `(%s)` would be enough. "HTTP" or "HTTPS" or "local" is enough context here IMO, and it keeps the logs a bit shorter.